ltiauthenticator/lti11/auth.py

```diff
@@ -59,6 +79,12 @@ async def authenticate(  # noqa: C901
         Raises:
             HTTPError if the required values are not in the request
         """
+        # log deprecation warning when using the default custom_canvas_user_id setting
+        if self.username_key == "custom_canvas_user_id":
+            self.log.warning(
+                "The default username_key 'custom_canvas_user_id' will be replaced by 'user_id' in a future release."
```

Comment:
```suggestion
                dedent("""The default username_key 'custom_canvas_user_id' will be replaced by 'user_id' in a future release.
                Set c.LTIAuthenticator.username_key to `custom_canvas_user_id` to preserve current behavior
                """)
```

Might have to `from textwrap import dedent`